src/Mount.cpp

```diff
@@ -1328,46 +1348,110 @@ void Mount::startSlewingToTarget()
 
 /////////////////////////////////
 //
-// stopGuiding
+// startSlewingToHome
 //
 /////////////////////////////////
-void Mount::stopGuiding()
+// Calculates movement parameters and program steppers to move to home position.
+// Takes any sync operations that have happened and tracking into account.
+void Mount::startSlewingToHome()
 {
-    stopGuiding(true, true);
+    stopGuiding();
+
+    // Make sure we're slewing at full speed on a GoTo
+    LOGV2(DEBUG_STEPPERS, F("[STEPPERS]: startSlewingToHome: Set DEC to MaxSpeed(%d)"), _maxDECSpeed);
+    _stepperDEC->setMaxSpeed(_maxDECSpeed);
+    LOGV2(DEBUG_STEPPERS, F("[STEPPERS]: startSlewingToHome: Set RA  to MaxSpeed(%d)"), _maxRASpeed);
+    _stepperRA->setMaxSpeed(_maxRASpeed);
+
+    _currentDECStepperPosition = _stepperDEC->currentPosition();
+    _currentRAStepperPosition  = _stepperRA->currentPosition();
+
+    // Take any syncs that have happened into account
+    long targetRAPosition  = -_homeOffsetRA;
+    long targetDECPosition = -_homeOffsetDEC;
+    LOGV3(DEBUG_STEPPERS, F("[STEPPERS]: startSlewingToHome: Sync op offsets: RA: %l, DEC: %l"), targetRAPosition, targetDECPosition);
+
+    _slewingToHome = true;
+    // Take tracking into account
+    long trackingOffset = _stepperTRK->currentPosition() * RA_SLEW_MICROSTEPPING / RA_TRACKING_MICROSTEPPING;
+    targetRAPosition -= trackingOffset;
+    LOGV4(DEBUG_STEPPERS,
+          F("[STEPPERS]: startSlewingToHome: Adjusted with tracking distance: %l (adjusted for MS: %l), result: %l"),
+          _stepperTRK->currentPosition(),
+          trackingOffset,
+          targetRAPosition);
+
+    moveSteppersTo(targetRAPosition, targetDECPosition);  // u-steps (in slew mode)
+
+    _mountStatus |= STATUS_SLEWING | STATUS_SLEWING_TO_TARGET;
+    _totalDECMove = 1.0f * _stepperDEC->distanceToGo();
```

Comment:
```C++
_totalDECMove = static_cast<float>(_stepperDEC->distanceToGo());
_totalRAMove  = static_cast<float>(_stepperRA->distanceToGo());
```
instead of the `1.0f *`